### PR TITLE
Fix no user name error not shown in console when the name is blank

### PR DESCRIPTION
**Pull request by juozaspo**
**(updated x2)** If a blank name or nothing were provided while running minetest, the game would not show any error in console, and exit if menu were skipped using --go parameter in command line. The line that was missing were added to display the error message in the console, extra check removed.

### CODE REVIEW

File: src/client/clientlauncher.cpp [hunk 1]
```diff
@@ -401,6 +401,8 @@ bool ClientLauncher::launch_game(std::string &error_message,
 
 	if (menudata.name == "" && !simple_singleplayer_mode) {
 		error_message = gettext("Please choose a name!");
+		if (skip_main_menu)
```
Comment (SmallJoker) [inline src/client/clientlauncher.cpp]: This check can be omitted. See below: The error message is always written to `errorstream` after setting its content.

**Comment by juozaspo** [comment]
**Update:** Changed last commit message through git client, text was a bit misleading.

File: src/client/clientlauncher.cpp (hunk 1 above)
Comment (juozaspo) [inline src/client/clientlauncher.cpp]: Done. Added a commit that removes the check.

**Comment by juozaspo** [comment]
changing source branch. will be right back on new PR since github doesn't allow change the branch you pull from.

**Comment by SmallJoker** [comment]
This PR was no problem to merge (easy as `wget https://github.com/minetest/minetest/pull/6214.patch && git am -3 6214.patch`) - it would just be a bit annoying for you to reset your local master to pull the new commit again.

**Comment by juozaspo** [comment]
The problem is, if I add new upstream commits by using **git merge** command and pushing to github master branch, the unrelated comits gets automatically added to the PR, as long as it's open. I don't want that, so I opened another PR, pointing to correct branch, keeping master up to date and unrelated development in a yet another local or remote branch.

**Pull request closed without merging**